commerce_coordinator/apps/rollout/pipeline.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import waffle
+from openedx_filters import PipelineStep
+
+from commerce_coordinator.apps.commercetools_frontend.constants import COMMERCETOOLS_FRONTEND
+from commerce_coordinator.apps.frontend_app_payment.constants import FRONTEND_APP_PAYMENT_CHECKOUT
+from commerce_coordinator.apps.rollout.constants import REDIRECT_TO_COMMERCETOOLS_CHECKOUT
+
+logger = logging.getLogger(__name__)
+
+
+class GetActiveOrderManagementSystem(PipelineStep):
+    """
+    Determines which order management system to redirect to, based on enterprise role.
+    """
+
+    def run_filter(self, request, active_om_system):
+        """
+        Execute a filter with the signature specified.
+        Arguments:
+            request: request object passed through from the lms filter
+            active_om_system: result from pipeline steps to determine if user has
+                            enterprise role or not & dictact which checkout to redirect to
+        """
+
+        sku = request.query_params.get('sku', '').strip()
+        sku_exists = isinstance(sku, str) and sku  # True if sku not None and is string
+        course_run = request.query_params.get('course_run_key', '').strip()
+        course_run_exists = isinstance(course_run, str) and course_run  # True if course_run_key not None and is string
+
+        if waffle.flag_is_active(request, REDIRECT_TO_COMMERCETOOLS_CHECKOUT) and sku_exists and course_run_exists:
+            active_om_system = COMMERCETOOLS_FRONTEND
+        else:
+            active_om_system = FRONTEND_APP_PAYMENT_CHECKOUT
```

Comment:
```suggestion
        elif sku:
            active_om_system = FRONTEND_APP_PAYMENT_CHECKOUT
        else:
            <log and raise error>
```

Reply:
If we are only checking `sku` here, is there a need for `sku_exists`?